resources/public/api/conf/3.0/schemas/retrieve_se_adjustments.json

```diff
@@ -96,14 +96,15 @@
             "type": "object",
             "properties": {
               "periodId": {
-                "description": "A reference to the unique identifier of the period. Must conform to the regular expression: \n\n YYYY-MM-DD_YYYY-MM-DD",
+                "description": "A reference to the unique identifier of the period. Must conform to: \\nYYYY-MM-DD_YYYY-MM-DD\"",
```

Comment:
can you remove the \n and " around YYYY-MM-DD_YYYY-MM-DD because it is showing up on the RAML preview